sim/priest/items.go

```diff
@@ -138,10 +175,15 @@ var ItemSetCrimsonAcolyte = core.NewItemSet(core.ItemSet{
 	Name: "Crimson Acolyte's Regalia",
 	Bonuses: map[int32]core.ApplyEffect{
 		2: func(agent core.Agent) {
-			// Implemented in vampiric_touch.go/devouring_plague.go/swp.go
+			character := agent.GetCharacter()
+			character.AddStaticMod(core.SpellModConfig{
+				Kind:       core.SpellMod_BonusCrit_Rating,
+				FloatValue: 5 * core.CritRatingPerCritChance,
+				ClassMask:  PriestSpellShadowWordPain | PriestSpellDevouringPlague | PriestSpellVampiricTouch | PriestSpellImprovedDevouringPlague,
+			})
 		},
 		4: func(agent core.Agent) {
-			// Implemented in mind_flay.go
+			// TODO: Implemement proper aura mod to mod total duration
```

Comment:
Spells have a field called `RelatedAuras` which you can assign aura references to be kept in the spell so we can add a spell mod for aura duration and modify the duration of all related auras of that spell

Reply:
Honestly I forgot about this place holder. The correct wording would be 'Shorten tick period' i guess. As each tick is shortened by 0.17 and we got 3 ticks so a total of 0.5 seconds reduction. Wasn't sure how to properly implement that but I could do the same as you did for the SpelMod_NumTicks i guess.

Reply:
Done.